invenio_app_ils/records/loaders/schemas/json/series.py

```diff
@@ -53,4 +70,27 @@ class Meta:
     pid = PersistentIdentifier()
     publisher = fields.Str()
     title = fields.Str(required=True)
+    updated_by = fields.Nested(ChangeBySchema)
     urls = fields.Nested(UrlSchema, many=True)
+
+    @pre_load
+    def add_created_by(self, data, **kwargs):
```

Comment:
If we need to add this even more times (other records), we might want to refactor in the future and put it in common so that we don't duplicate code.